tasks/PageIteratorTask.md

```diff
@@ -125,14 +125,51 @@ pageIterator.Iterate(func(pageItem interface{}) bool {
 })
 ```
 
+PHP
+```php
+$pageIterator = new PageIterator($graphResponse, $requestAdapter, $parsableConstructor);
+$pageIterator->setHeaders(["Content-Type" => "application/json"]);
+
+$items = [];
+$pageIterator->iterate(function ($pageItem) use (&$items) {
+      $deserializer = new JsonParseNode($pageItem);
+      /** var User $item **/
+      $item = $deserializer->getObjectValue([User::class, 'createFromDiscriminatorValue']);
+      $items []= $item;
+      return true;
```

Comment:
I don't think we should expect the page iterator consumer to deserialize models themselves, this is something we should be doing for them.
Granted they might have to downcast if the generic types support in PHP is not very advanced